run/run.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/**
+ * @description undefined
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+async function _command(params, commandText, secrets = {}) {
+  const {mmURL, mmToken} = secrets;
+  if (!mmURL || !mmToken) {
+    return {
+      response_type: 'ephemeral',
+      text: `You need to create secrets named \`mmURL\` & \`mmToken\` with your mattermost server URL & API token. Create secrets by running \`/nc secret_create\``
```

Comment:
Split this into multiple lines.

Reply:
Will mark ready for review soon. Need to fix quite a few things.